docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -120,12 +124,31 @@ Updates to indicate you're running your experimental version. You can uninstall
 your version and go back to the originally installed version by choosing your
 version and clicking Uninstall.
 
+### Deploying with command-line
+
+You can build and deploy with the following command: 
+`.\Build.cmd -Configuration Release -deployExtensions -launch`.
```

Comment:
Is there anywhere to find docs for `Build.cmd` args?

Reply:
`.\Build.cmd -help` gives:

```
Common settings:
  -configuration <value>    Build configuration: 'Debug' or 'Release' (short: -c)
  -verbosity <value>        Msbuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic]
  -deployExtensions         Deploy built vsixes (short: -d)
  -binaryLog                Create MSBuild binary log (short: -bl)
  -buildServerLog           Create Roslyn build server log

Actions:
  -restore                  Restore packages (short: -r)
  -build                    Build main solution (short: -b)
  -rebuild                  Rebuild main solution
  -pack                     Build NuGet packages, VS insertion manifests and installer
  -sign                     Sign our binaries
  -publish                  Publish build artifacts (e.g. symbols)
  -launch                   Launch Visual Studio in developer hive
  -help                     Print help and exit

Test actions
  -test32                   Run unit tests in the 32-bit runner
  -test64                   Run units tests in the 64-bit runner
  -testDesktop              Run Desktop unit tests (short: -test)
  -testCoreClr              Run CoreClr unit tests
  -testVsi                  Run all integration tests
  -testIOperation           Run extra checks to validate IOperations

Advanced settings:
  -ci                       Set when running on CI server
  -bootstrap                Build using a bootstrap compilers
  -bootstrapConfiguration   Build configuration for bootstrap compiler: 'Debug' or 'Release'
  -msbuildEngine <value>    Msbuild engine to use to run build ('dotnet', 'vs', or unspecified).
  -procdump                 Monitor test runs with procdump
  -skipAnalyzers            Do not run analyzers during build operations
  -prepareMachine           Prepare machine for CI run, clean up processes after build
  -useGlobalNuGetCache      Use global NuGet cache.
  -warnAsError              Treat all warnings as errors
  -sourceBuild              Simulate building source-build

Official build settings:
  -officialBuildId                            An official build id, e.g. 20190102.3
  -officialSkipTests <bool>                   Pass 'true' to not run tests
  -officialSkipApplyOptimizationData <bool>   Pass 'true' to not apply optimization data
  -officialSourceBranchName <string>          The source branch name
  -officialIbcDropId <string>                 IBC data drop to use (e.g. '20190210.1/935479/1').
                                              'default' for the most recent available for the branch.
  -officialIbcSourceBranchName <string>       IBC source branch (e.g. 'master-vs-deps')
                                              'default' to select branch based on eng/config/PublishData.json.
```

Reply:
Hum, it looks like the `-launch` option isn't working... 
Tagging @sharwell as FYI. Let me know if you need a tracking issue.

Reply:
I use `-launch` frequently.

Reply:
Retried and `-launch` worked. Not sure how I got confused...
@sharwell Could I have a sign-off so that I can merge this doc-only change?****